### DISCUSSION TITLE
Converting from Mark up Editor to Visual Editor resets Block Quotes

### DISCUSSION THREAD

**Discussion by darashayda** [requarks/wiki#6178] [Error / Bug Report] [answered]
Converting from Markup Editor to Visual Editor causes all Block Quotes to become the default grey one!

**Comment by NGPixel** [comment, marked as answer]
Not all features are supported when converting between editors. Use the Markdown editor if you need styled blockquotes.

**Reply by darashayda** [reply]
Thanks, figured that much out, while I have you in person, could you please tell me how to use the Stiky or floating table for the headers, it was there and you had mentioned in some forums that it was provided. 

Thanx
